examples/aya-tool/xtask/src/main.rs

```diff
@@ -3,21 +3,21 @@ mod run;
 
 use std::process::exit;
 
-use structopt::StructOpt;
-#[derive(StructOpt)]
+use clap::Parser;
+#[derive(Debug, Parser)]
```

Comment:
Wait, is it generated like that? Without an empty line in between the import and `derive`? :scream: 

If so, we need to fix that. I remember you got it working. Did we forget to fix that properly also in xtask?

Reply:
Nah, it looks fine in the template

https://github.com/aya-rs/aya-template/blob/main/xtask/src/main.rs#L7

Please fix it here

Reply:
done